src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,45 @@
+.stopwatch {
+  display: flex;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  &__center {
+    position: absolute;
+    z-index: 1;
+    height: 30px;
+    width: 30px;
+    background-color: #4b9aaa;
+    border-radius: 50%;
+  }
+
+  &__hand {
+    position: absolute;
+    width: 12px;
+    border-radius: 6px;
+
+    &--second {
+      height: 30%;
+      background-color: #d15559;
+      bottom: 50%;
+      transform-origin: bottom;
+      animation-name: move;
+      animation-duration: 60s;
+      animation-timing-function: steps(60);
+      animation-iteration-count: infinite;
```

Comment:
If you use such form of writing animation you can use mixin or placeholder to reduce code's strings